tests/test_meshtastic_utils_connect_paths.py

```
@@ -23,6 +25,17 @@
 from mmrelay.meshtastic_utils import connect_meshtastic, on_lost_meshtastic_connection
 
 
+def _schedule_reconnect_closing_coro(
+    coro: Coroutine[Any, Any, Any],
+    loop: asyncio.AbstractEventLoop | None = None,
```

Ruff (0.15.10)

[warning] Unused function argument: `loop`

(ARG001)